feddlib/core/FE/FE_def.hpp

```diff
@@ -7319,6 +7319,73 @@ void FE<SC,LO,GO,NO>::assemblyRHS( int dim,
     }
 }
 
+/// @brief Assembling \int p \dx = 0. Thus, we need the integral part for the mean pressure value. We need this to be in matrix format, as it is added to the system
```

Comment:
Ich dachte, das müsste in Frosch implementiert werden ?`

Reply:
Ja es kann sein, dass wir a zumindest in der FEDDLib assemblieren und dann übergeben. Habe es zur Sicherheit mal drin gelassen